api/provider/aws/load_balancer_create.go

```diff
@@ -211,6 +283,41 @@ func (l *LoadBalancerProvider) createLoadBalancer(
 	return nil
 }
 
+func (l *LoadBalancerProvider) createApplicationLoadBalancer(
+	loadBalancerName string,
+	scheme string,
+	securityGroupIDs []string,
+	subnetIDs []string,
+) (*alb.LoadBalancer, error) {
+	securityGroups := make([]*string, len(securityGroupIDs))
+	for i, securityGroupID := range securityGroupIDs {
+		securityGroups[i] = aws.String(securityGroupID)
```

Comment:
I think you're running into a `range` quirk here that I ran into a couple of weeks ago, where `securityGroups` will end up just being a slice of identical pointers to the final element of `securityGroupIDs`. The value pointer doesn't track all the way through the loop; it just catches the last value.

Example: https://play.golang.org/p/o0F_7HLWf0p